src/blogtoblog/BlogToBlogImporter.ts

```diff
@@ -102,7 +102,7 @@ export default class BlogToBlogImporter extends PageImporter {
       authorTitle.textContent = 'Author';
       authorRow.append(authorTitle);
       const authorData = document.createElement('td');
-      authorData.textContent = author;
+      authorData.textContent = author.trim();
```

Comment:
Nitpicking: why adding the `trim()` here and not when setting the `author` value ? There is already a `replace()` there...